### PR TITLE
port framework_agreement_sourcing to 8.0

### PR DESCRIPTION
**Pull request by lepistone**
Dependencies:
- [x] merge https://github.com/OCA/vertical-ngo/pull/8
- [x] merge https://github.com/OCA/purchase-workflow/pull/34

Checklist:
- [x] make tests pass in v8
- [x] lint
- [x] functional test

### CODE REVIEW

**Comment by yvaucher** [comment]
Warning:

```
File ".../vertical-ngo/framework_agreement_sourcing/model/logistic_requisition_cost_estimate.py", line 93, in cost_estimate
      if x and x.logistic_requisition_source_id.procurement_method
  AttributeError: 'sale.order.line' object has no attribute 'logistic_requisition_source_id'
```

logistic_requisition_source_id was removed from sale.order

**Comment by lepistone** [comment]
@yvaucher well spotted, you're right. That method needs to be adapted then, let's talk about it when you want. Thanks!

File: framework_agreement_sourcing/model/logistic_requisition_cost_estimate.py [hunk 1]
```diff
@@ -24,11 +24,11 @@
 
 
 class logistic_requisition_cost_estimate(orm.Model):
```
Comment (yvaucher) [inline framework_agreement_sourcing/model/logistic_requisition_cost_estimate.py]: Should certainly be a TransientModel

**Comment by lepistone** [comment]
Fixed as suggested by @yvaucher and rebased.

**Comment by lepistone** [comment]
I checked "functional testing". More testing would help for sure, but I think we can merge it.

**Comment by yvaucher** [comment]
@lepistone can you rebase to relaunch travis ?

**Comment by lepistone** [comment]
We're :green_heart: :green_apple:

File: framework_agreement_sourcing/__openerp__.py [hunk 2]
```diff
@@ -34,8 +34,8 @@
 agreement will be used as source for the concerned source lines
 of your request.
 
-In this case tender flow is byassed and confirmed PO will be generated
-when logistic requisition is confirmed.
+In this case the tender flow is bypassed and confirmed PO will be generated
+when the logistic requisition is confirmed.
```
Comment (yvaucher) [inline framework_agreement_sourcing/__openerp__.py:38]: In description bellow this comment:
s/generating/generated/
s/currences/currencies/

**Comment by yvaucher** [comment]
appart from non blocking typo :+1:

**Comment by lepistone** [comment]
typo fixed.

**Comment by yvaucher** [comment]
:+1: thx

**Comment by gurneyalex** [comment]
this broke the build probably because depending on #14

**Pull request merged**